design-dimensions/Dimensions-with-General-Agreement.md

```diff
@@ -25,6 +25,11 @@ We’ve explored three main definitions of privacy:
 
 The community group has reached general agreement that the _Private Measurement Technical Specification MVP_ should use a definition of privacy based on differential privacy. This does not preclude the use of other privacy definitions in conjunction with differential privacy, however any proposal should aim to provide differential privacy guarantees.
 
+### Private measurement of single events
+The group has discussed the use-case of “differentially private measurement of single events” i.e. whether, under a differential privacy guarantee, we would permit queries which ask about a single event’s outcome (did impression lead to a conversion, for instance). For more information about this, see https://github.com/patcg/docs-and-reports/issues/41 and the [agenda topic](https://github.com/patcg/meetings/issues/112) from the May 2023 meeting.
+
+We have reached a general consensus that these types of queries can be particularly sensitive, so we must take care when setting the proper [differential privacy scope](#privacy-unit--privacy-budget-scoping) as well as privacy parameters (e.g. epsilon) to ensure that events are properly protected, and we ensure that the protection covers people that submit many events. However, queries of this form can be generally permitted, and we do not require special mitigations to ensure aggregation across many events is happening.
```

Comment:
```suggestion
We have reached consensus that these types of queries can be particularly sensitive, so we must take care when setting the proper [differential privacy scope](#privacy-unit--privacy-budget-scoping) and parameters (e.g., epsilon and delta) to ensure that user contributions are adequately protected. Any protection needs to cover people that submit many events. However, queries of this form will be considered.
```

I might be in the rough with this last point, but I'm not giving up on requiring additional protections just yet.  Yes, our analytic framework does not give us the tools necessary to obtain any surety about the results being for more than one person.  That is an excuse, not justification.

Reply:
> I'm not giving up on requiring additional protections just yet

Is this coming from a concern that DP guarantees will be too weak? I tried to phrase this so that essentially we could agree that plugging in _some_ epsilon could satisfy us. What do you think about something like the following for the last sentence, which carves out mitigations in case we aren't satisfied with the DP.

```suggestion
We have reached consensus that these types of queries can be particularly sensitive, so we must take care when setting the proper [differential privacy scope](#privacy-unit--privacy-budget-scoping) and parameters (e.g., epsilon and delta) to ensure that user contributions are adequately protected. Any protection needs to cover people that submit many events. However, queries of this form will not necessarily be prohibited, although additional protections should be considered if differential privacy guarantees are deemed too weak.
```

I think this ensures that we don't "overly rely" on a DP bound that we don't believe offers enough protection and leaves room for researching mitigations.

Reply:
That works for me.